packages/block-editor/src/hooks/position.scss

```diff
@@ -1,18 +0,0 @@
-.block-editor-hooks__position-selection__select-control {
-	.components-custom-select-control__hint {
-		display: none;
-	}
-}
-
-.block-editor-hooks__position-selection__select-control__option {
-	&.has-hint {
-		grid-template-columns: auto 30px;
-		line-height: $default-line-height;
-		margin-bottom: 0;
-	}
-
-	.components-custom-select-control__item-hint {
-		grid-row: 2;
-		text-align: left;
-	}
-}
```

Comment:
These overrides shouldn't be necessary anymore because the V2 legacy adapter already allows the item hint to render on a separate line when there isn't enough space and sets the correct line height